recipes/packio/all/conanfile.py

```diff
@@ -9,33 +9,55 @@ class PackioConan(ConanFile):
     license = "MPL-2.0"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/qchateau/packio"
-    description = "An asynchronous msgpack-RPC library built on top of Boost.Asio."
-    topics = ("rpc", "msgpack", "asio", "async", "cpp17", "cpp20", "coroutines")
+    description = "An asynchronous msgpack-RPC and JSON-RPC library built on top of Boost.Asio."
+    topics = ("rpc", "msgpack", "json", "asio", "async", "cpp17", "cpp20", "coroutines")
     settings = "compiler"
     no_copy_source = True
-    options = {"standalone_asio": [True, False]}
-    default_options = {"standalone_asio": False}
+    options = {
+        "standalone_asio": [True, False],
+        "msgpack": [True, False],
+        "nlohmann_json": [True, False],
+    }
+    default_options = {
+        "standalone_asio": False,
+        "msgpack": True,
+        "nlohmann_json": True,
+    }
 
     @property
     def _source_subfolder(self):
         return "source_subfolder"
 
     def _supports_cpp17(self):
-        supported_compilers = [("apple-clang", 10), ("clang", 6), ("gcc", 7), ("Visual Studio", 16)]
-        compiler, version = self.settings.compiler, Version(self.settings.compiler.version)
+        supported_compilers = [
+            ("apple-clang", 10),
+            ("clang", 6),
+            ("gcc", 7),
+            ("Visual Studio", 16),
+        ]
+        compiler, version = self.settings.compiler, Version(
+            self.settings.compiler.version
+        )
         return any(compiler == sc[0] and version >= sc[1] for sc in supported_compilers)
```

Comment:
Could you remove this pattern please? It prevents unknown compilers to consume this lib. Replace with this king of checks: https://github.com/conan-io/conan-center-index/blob/4ac63884d528d6abb2d4740acb88d5cbd3d6b76a/recipes/spy/all/conanfile.py#L31

Reply:
done